contracts/events.md

```diff
@@ -12,7 +11,7 @@ The events are stored in the [contract call object](../serializations.md#contrac
 and can be accessed by looking up the transaction in which they were emitted. 
 
 Let's consider the following Sophia example:
-```
+```sophia
```

Comment:
I doubt github highlighter supports that, if you find a closer to syntax language that would be better isn't it ? If not, this is better than missing annotation I guess.

Reply:
well I hope I can push this topic forward. there is a quite old PR about this. so I wanted at least to prepare the code, see https://github.com/github/linguist/pull/4719.

we need to revisit this.